src/Workspaces/Core/Portable/Storage/SQLite/v2/SQLitePersistentStorage_FlushWrites.cs

```diff
@@ -29,47 +29,17 @@ private async ValueTask FlushInMemoryDataToDiskIfNotShutdownAsync(CancellationTo
         await PerformWriteAsync(_flushInMemoryDataToDisk, cancellationToken).ConfigureAwait(false);
     }
 
-    private Task FlushWritesOnCloseAsync()
```

Comment:
this intentionally went away.  There is an ABWQ that is already flushign writes to disk. That will still continue working up until the point that the disposal token fires (which means VS is shutting down).  We've always had the semantics that if we are shutting down, we strictly shut down, potentially losing some writes to disk (note: at worse we only ever lose writes in the las 0.5 seconds before shutdown).

Tthis is always safe to then skip.  If we have any final writes, they'll either get flushed or not.  If they do, great.  If they don't, that's fine as well.  That's because the persistent cache is *only* ever considered 'best effort'.  100% of featues taht use it can only use it as a place they try to store data into, but work 100% fine if those writes didn't happen.   This will be detected the next tmie VS runs when a feature tries to erad the data, and simply goes "the data isn't htere, so i'll compute it".